packs/data/pathfinder-bestiary-3.db/hadrinnex.json

```diff
@@ -120,6 +111,162 @@
     },
     "img": "systems/pf2e/icons/default-icons/npc.svg",
     "items": [
+        {
+            "_id": "fmeQBNnhZYs7wjnO",
+            "data": {
+                "badge": null,
+                "description": {
+                    "value": "<p>The creature gains physical damage resistance similar to the triggering type.</p>"
+                },
+                "duration": {
+                    "expiry": "turn-start",
+                    "sustained": false,
+                    "unit": "unlimited",
+                    "value": -1
+                },
+                "level": {
+                    "value": 4
+                },
+                "rules": [
+                    {
+                        "choices": [
+                            {
+                                "label": "PF2E.TraitBludgeoning",
+                                "value": "bludgeoning"
+                            },
+                            {
+                                "label": "PF2E.TraitPiercing",
+                                "value": "piercing"
+                            },
+                            {
+                                "label": "PF2E.TraitSlashing",
+                                "value": "slashing"
+                            }
+                        ],
+                        "flag": "physical-damage",
+                        "key": "ChoiceSet",
+                        "selection": "bludgeoning"
+                    },
+                    {
+                        "key": "Resistance",
+                        "type": "{item|flags.pf2e.rulesSelections.physical-damage}",
+                        "value": 15
+                    }
+                ],
+                "slug": null,
+                "source": {
+                    "value": ""
+                },
+                "start": {
+                    "initiative": null,
+                    "value": 126
+                },
+                "target": null,
+                "tokenIcon": {
+                    "show": true
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                }
+            },
+            "flags": {
+                "core": {
+                    "sourceId": "Item.E0YoQTMkZWLEk5HB"
+                }
+            },
```

Comment:
```suggestion
            },
```

Reply:
Hm, actually not sure, why this got in. Actually, this is an effect item from the bestiary effects compendium. That said - remove the source entirely or change it to the id for the compendium item?